aos_updatemanager.cfg

```diff
@@ -2,9 +2,10 @@
     "ID": "um",
     "ServerUrl": "localhost:9000",
     "CACert": "/etc/ssl/certs/rootCA.crt",
-    "CertStorage": "/var/aos/crypt/um/",
+    "CertStorage": "um",
```

Comment:
Move changes retated CertStorage to separate commit

Reply:
Done